speech/grpc/src/main/proto/google/longrunning/operations.proto

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) 2015, Google Inc.
```

Comment:
Does this really need to be here?  Shouldn't it be part of the gcloud or some other api repo.

If you are going to have these, then you should put them in third_party/ as they are copies of published code.  You should talk w/ @tswast about git magic for the include as well.